lib/node_runner_isaac_test.go

```diff
@@ -292,8 +299,29 @@ func TestNodeRunnerConsensusSameSourceWillNotIgnored(t *testing.T) {
 		wg.Done()
 	}
 
+	var secondHandleBallotCheckerFuncs = []sebakcommon.CheckerFunc{
+		CheckNodeRunnerHandleBallotIsWellformed,
+		CheckNodeRunnerHandleBallotCheckIsNew,
+		CheckNodeRunnerHandleBallotReceiveBallot,
+		CheckNodeRunnerHandleBallotHistory,
+		// CheckNodeRunnerHandleBallotStore
+		func(c sebakcommon.Checker, args ...interface{}) (err error) {
```

Comment:
This func seems like just doing return but doing nothing. Is it necessary?

Reply:
I guess that  I get what you mean.  :-)  Although the test code works without it(the empty func), it exists for to point to `CheckNodeRunnerHandleBallotStore`. Is it right?

Reply:
@anarcher Without this empty function, test will not be passed. :)

Reply:
Really?  I thought this test will be passed without the empty func :-)

Reply:
@anarcher Oh, yes, you right. You reminds me I'm stupid ㅋㅋㅋ

Reply:
Never! HAHA.  IMO,  The empty function represents that `CheckNodeRunnerHandleBallotStore` is empty. So I would like to keep this empty function. :-D